odbc_fdw.c

```diff
@@ -517,7 +527,13 @@ sql_data_type(
 		   appendStringInfo(sql_type, "float8");
 			break;
 		case SQL_BIT :
-		    appendStringInfo(sql_type, "bit(1)");
+			/* Use boolean instead of bit(1) because:
+			 * * binary types are not yet fully supported
+			 * * boolean is more commonly used in PG
+			 * * With options BoolsAsChar=0 this allows
+			 *   preserving boolean columns from pSQL ODBC.
+			 */
+		    appendStringInfo(sql_type, "boolean");
```

Comment:
I think that misalignment is due to how github handles tabs, but I sure do need a linter